doc/api/fs.md

```diff
@@ -2745,12 +2745,31 @@ changes:
                  it will emit a deprecation warning.
 -->
 
-* `path` {string|Buffer|URL}
+* `path` {string|Buffer|URL} File name or file descriptor
 * `callback` {Function}
   * `err` {Error}
 
-Asynchronous unlink(2). No arguments other than a possible exception are given
-to the completion callback.
+Asynchronously removes a file or symbolic link. No arguments other than a
+possible exception are given to the completion callback.
+
+```js
+// Assuming that 'path/file.txt' is a regular file.
+fs.unlink('path/file.txt', (err) => {
+  if (err) throw err;
+  console.log('path/file.txt was deleted');
+});
+```
+
+Note that unlink() will not work on a directory, empty or otherwise.
+
+```js
+// Assuming that 'tmpDir' is a directory.
+fs.unlink('tmpDir', (err) => {
+  // err is not null because a directory cannot be deleted through unlink().
+});
+```
+
+See also: unlink(2)
```

Comment:
I know this already occurs in six other places in the doc, but I have to say: A "see also" without a link or something is a bit user-hostile. I mean, we know this is talking about POSIX `unlink()` in man page section 2, but is it really fair to expect everyone to understand that?

Reply:
(The comment above is not a request to change this here. It's more of a flagging of a larger issue that is for a bigger scope than this PR.)

Reply:
Oh, looks like my "larger issue" comment is invalid because this stuff *does* end up as a link once it gets published to the website. I'll delete this comment and the preceding two. Sorry about the noise!